content-src/experiments/send.yaml

```diff
@@ -0,0 +1,82 @@
+id: 13
+title: 'Send'
+slug: send
+platforms: ['web']
+web_url: 'https://send.firefox.com'
+thumbnail: /static/images/experiments/send/experiments_experiment/thumbnail.png
+description: >
+  Encrypt and send files with a link that automatically expires to ensure your important documents don’t stay online forever.
+introduction: >
+  <p>Send lets you upload and encrypt large files (up to 1GB) to share online. When you upload a file, send creates a link to pass along to whoever you want. Each link created by Send will expire after 1 download or 24 hours, and all sent files will be automatically deleted from the Send Server.</p>
+  <p>Unlike other Test Pilot experiments. Send does not require an add-on, and can be used in any modern browser.</p>
+image_twitter: /static/images/experiments/send/social/send-twitter.jpg
+image_facebook: /static/images/experiments/send/social/send-fb.jpg
+changelog_url: 'https://github.com/mozilla/send/master/CHANGELOG.md'
```

Comment:
Nit: we should change this to https://github.com/mozilla/send/releases until we have a CHANGELOG.md file in that repo.